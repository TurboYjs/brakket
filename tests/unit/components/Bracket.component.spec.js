import { shallow } from "@vue/test-utils"
import Bracket from "@/components/Bracket.component.vue"

describe("Bracket.component.vue", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(Bracket)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
