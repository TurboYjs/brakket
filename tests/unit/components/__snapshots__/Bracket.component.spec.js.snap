// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bracket.component.vue matches snapshot 1`] = `
<div>
    Bracket
</div>
`;
